sim-cli/src/parsing.rs

```diff
@@ -123,6 +123,13 @@ impl Cli {
             ));
         }
 
+        if sim_params.sim_network.is_empty() && !sim_params.exclude.is_empty() {
+            return Err(anyhow!(
+                "List of nodes to exclude from sending/receiving
+                    in random activity is only allowed on a simulated network"
```

Comment:
nit: note that user can just remove a node from `nodes` if they don't want it included for regular operation